apps/epic-react/.env.development

```diff
@@ -31,7 +31,9 @@ EMAIL_SERVER_PORT=587
 
 # ConvertKit Newsletter Form
 NEXT_PUBLIC_CONVERTKIT_SIGNUP_FORM=FORM_ID
-NEXT_PUBLIC_CONVERTKIT_TOKEN=CK_TOKEN
+NEXT_PUBLIC_CONVERTKIT_REACT_SIGNUP_FORM=FORM_ID
+NEXT_CONVERTKIT_TOKEN=CK_TOKEN
+NEXT_PUBLIC_CONVERTKIT_BASE_URL=BASE_URL
 NEXT_PUBLIC_CONVERTKIT_ANSWER_URL=/api/skill/answer/convertkit
```

Comment:
wasn't sure if the form id or base url were sensitive (base url probably not?) but put them in here as env vars